app/views/devise/sessions/new.html.erb

```diff
@@ -50,16 +50,9 @@
       ) %>
   <%= f.submit t('links.next'), full_width: true, wide: false %>
   <% if @sign_in_a_b_test_bucket == :default %>
-    <h2 class='separator-text'>
-      <%= t('headings.create_account_with_sp.cta', app_name: APP_NAME) %>
-    </h2>
-    <%= render ButtonComponent.new(
-          action: ->(**tag_options, &block) { link_to(sign_up_email_url(source: :sign_in), **tag_options, &block) },
-          big: true,
-          outline: true,
-          full_width: true,
-          class: 'margin-bottom-105',
-        ).with_content(t('links.create_account')) %>
+    <div class="margin-top-4 margin-bottom-2">
+      <%= render 'shared/create_an_account_banner' %>
+    </div>
```

Comment:
This condition is for the default variant, but it's rendering the banner?

Reply:
Ok I think I was confused by the partial name. The partial is named banner, but it's actually the default button?

Reply:
I named the partial the way I did because I went with the purpose of the banner and where it should be located. So if the test is running, put it in the location where the test is being run vs. the current state.